usecases/connect-autonomous-database/src/main/java/com/oracle/cloud/world/demo/db/DBSettings.java

```diff
@@ -0,0 +1,46 @@
+package com.oracle.cloud.world.demo.db;
```

Comment:
Suggested to remove Cloud World reference (generalize)...

Reply:
Renamed the package name